integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiJmsRecovery.java

```diff
@@ -0,0 +1,739 @@
+        adminServerPodName, domainNamespace);
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
+    // create the required leasing table 'ACTIVE' before we start the cluster
+    createLeasingTable();
+    // check managed server services and pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server services and pods are created in namespace {0}",
+          domainNamespace);
+      checkPodReadyAndServiceExists(managedServerPrefix + i, domainUid, domainNamespace);
+    }
+  }
+
+  /**
+   * Verify all server pods are running.
+   * Verify all k8s services for all servers are created.
+   */
+  @BeforeEach
+  public void beforeEach() {
+    logger.info("Running beforeEach() ....");
+  }
+
+  /**
+   * Verify JMS Service is migrated to an available active server.
+   * Here the JMS messages are stored in File store on PV
+   */
+  @Test
+  @Order(1)
+  @DisplayName("Verify JMS Service migration with FileStore")
+  public void testMiiJmsServiceMigrationWithFileStore() {
+
+    buildClientOnPod();
+    runJmsClientOnAdminPod("send", 
+            "ClusterJmsServer@managed-server2@jms.testUniformQueue");
+
+    boolean psuccess = assertDoesNotThrow(() ->
+            scaleCluster(domainUid, domainNamespace, "cluster-1", 1),
+        String.format("replica patching to 1 failed for domain %s in namespace %s", domainUid, domainNamespace));
+    assertTrue(psuccess,
+        String.format("Cluster replica patching failed for domain %s in namespace %s", domainUid, domainNamespace));
+    checkPodDoesNotExist(managedServerPrefix + "2", domainUid, domainNamespace);
+    runJmsClientOnAdminPod("receive", 
+            "ClusterJmsServer@managed-server2@jms.testUniformQueue");
+  }
+
+  /**
+   * Verify JMS Service is migrated to an available active server.
+   * Here the JMS messages are stored in JDBC store.
+   */
+  @Test
+  @Order(2)
+  @DisplayName("Verify JMS Service migration with JDBCStore")
+  public void testMiiJmsServiceMigrationWithJdbcStore() {
+
+    // Restart the cluster, so that JMS server runtime are hosted 
+    // on the respective managed server
+    restartCluster();
+    buildClientOnPod();
+
+    runJmsClientOnAdminPod("send", 
+            "JdbcJmsServer@managed-server2@jms.jdbcUniformQueue");
+    boolean psuccess3 = assertDoesNotThrow(() ->
+            scaleCluster(domainUid, domainNamespace, "cluster-1", 1),
+        String.format("replica patching to 1 failed for domain %s in namespace %s", domainUid, domainNamespace));
+    assertTrue(psuccess3,
+        String.format("Cluster replica patching failed for domain %s in namespace %s", domainUid, domainNamespace));
+    checkPodDoesNotExist(managedServerPrefix + "2", domainUid, domainNamespace);
+
+    runJmsClientOnAdminPod("receive", 
+            "JdbcJmsServer@managed-server2@jms.jdbcUniformQueue");
+  }
+
+  // Create leasing Table (ACTIVE)
+  private static void createLeasingTable() {
```

Comment:
add this to DBUtils

Reply:
This method is specific to Service Migration usecase not a common utility to create any generic table.

Reply:
Move the method to common/DBUtils.java